eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -1,20 +1,31 @@
 {
+  "displayNames": {
+    "--disablecov": ""
+  },
   "matrix": {
     "Agent": {
-      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" },
-      "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" },
-      "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
+      "ubuntu-18.04": { "OSVmImage": "MMSUbuntu18.04", "Pool": "azsdk-pool-mms-ubuntu-1804-general" }
     },
-    "PythonVersion": [ "3.5", "3.7", "2.7", "pypy3" ],
-    "CoverageArg": [ "--disablecov" ]
+    "PythonVersion": [ "3.7", "3.8", "3.9", "pypy3" ],
```

Comment:
We don't care about linux 2.7 support?

Reply:
It kind of feels like you are trying to manually create a sparse matrix.

Reply:
I'll add 2.7 linux back in. The way python had things set up was testing the majority of versions against Linux, and then adding one case each to cover windows and mac, so that's how I have the matrix config set up here.